Sources/PointFree/SubscribeConfirmation.swift

```diff
@@ -220,39 +219,27 @@ func redirectActiveSubscribers<A>(
     return { conn in
       let user = user(conn.data)
 
-      let userSubscription =
-        (user?.subscriptionId)
-        .map { Current.database.fetchSubscriptionById($0).mapExcept(requireSome) }
-        ?? throwE(unit)
-
-      let ownerSubscription =
-        (user?.id)
-        .map { Current.database.fetchSubscriptionByOwnerId($0).mapExcept(requireSome) }
-        ?? throwE(unit)
-
-      let race = (userSubscription.run.parallel <|> ownerSubscription.run.parallel).sequential
-
-      return EitherIO(run: race)
```

Comment:
I removed this race for a simpler cascade, which is more deterministic. We actually have a test that fails about 7% of the time because of this non-determinism. Prob don't need to race our own DB anyway.